packages/config/src/projects/sischain/sischain.ts

```diff
@@ -0,0 +1,46 @@
+import { ProjectId, UnixTime } from '@l2beat/shared-pure'
+import type { ScalingProject } from '../../internalTypes'
+import { underReviewL2 } from '../../templates/underReview'
+
+export const sischain: ScalingProject = underReviewL2({
+  id: ProjectId('sischain'),
+  addedAt: UnixTime(1753778036),
+  capability: 'universal',
+  display: {
+    name: 'SIS Chain',
+    slug: 'sischain',
+    description:
+      'The SIS Chain is purpose-built for cross-chain activity – with fast execution, simple tooling, and native token utility at its core.',
+    category: 'ZK Rollup',
```

Comment:
why zk rollup? i would assume than an orbit (arbitrum) stack would be optimistic